### PR TITLE
5490 previous benefits sponsor file number

### PR DESCRIPTION
**Pull request by cvalarida**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1929
Requires https://github.com/department-of-veterans-affairs/vets-json-schema/pull/81 to be merged first

image

### CODE REVIEW

File: test/edu-benefits/5490/config/benefitHistory.unit.spec.jsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ describe('Edu 5490 benefitHistory', () => {
 
     // Should expand to 16
     expect(Array.from(formDOM.querySelectorAll('input,select')).length)
-      .to.equal(16);
+      .to.equal(17);
```
Comment (jbalboni) [inline test/edu-benefits/5490/config/benefitHistory.unit.spec.jsx]: I think we should also have a test that checks the required states for those two fields change appropriately.

**Review by jbalboni — APPROVED** [review]
LGTM, just had one request for another unit test.

File: test/edu-benefits/5490/config/benefitHistory.unit.spec.jsx (hunk 1 above)
Comment (cvalarida) [inline test/edu-benefits/5490/config/benefitHistory.unit.spec.jsx]: Isn't that a test somewhere else?

Comment (jbalboni) [inline test/edu-benefits/5490/config/benefitHistory.unit.spec.jsx]: There might be a similar test somewhere else, but lines 330 and 337 in the config above are new and wouldn't be covered by a test yet.

**Pull request merged**